package.json

```diff
@@ -1,41 +1,36 @@
 {
   "devDependencies": {
+    "@types/shelljs": "^0.7.8",
     "eslint": "4.13.1",
     "eslint-config-prettier": "2.9.0",
     "istanbul": "1.1.0-alpha.1",
     "lerna": "2.4.0",
     "prettier": "1.6.0",
     "remap-istanbul": "^0.9.5",
+    "shelljs": "^0.8.1",
     "shx": "0.2.2",
     "tslint": "5.5.0",
     "typescript": "2.6.2",
     "vsce": "1.29.0"
   },
   "scripts": {
-    "postinstall":
-      "lerna bootstrap -- --no-package-lock && node scripts/reformat-with-prettier",
-    "bootstrap":
-      "lerna bootstrap -- --no-package-lock && node scripts/reformat-with-prettier",
+    "postinstall": "lerna bootstrap -- --no-package-lock && node scripts/reformat-with-prettier",
```

Comment:
Are these changes because of `npm install`? I thought reformat-with-prettier.js fixes it: https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/scripts/reformat-with-prettier.js#L15

Reply:
I see the same diffs in package.json when I run npm install and end up removing my "changed" file before I push.

Reply:
I think a reformat-with-prettier.js is necessary here.

Reply:
@vazexqi Should we do automatic formatting or validation with a Git pre-commit hook? `typicode/husky` could help here.